### PR TITLE
Remove obsolete built in macros and add a `keccak256` macro alias for `sha3`.

### PR DESCRIPTION
**Pull request by zigguratt**

### CODE REVIEW

File: liblll/CompilerState.cpp [hunk 1]
```diff
@@ -45,8 +45,6 @@ CodeFragment const& CompilerState::getDef(std::string const& _s)
 void CompilerState::populateStandard()
 {
 	static const string s = "{"
-	"(def 'gav 0x51ba59315b3a95761d0863b05ccc7a7f54703d99)"
-	"(def 'config 0x661005d2720d855f1d9976f88bb10c1a3398c77f)"
 	"(def 'allgas (- (gas) 21))"
```
Comment (chriseth) [inline liblll/CompilerState.cpp:48]: Just a comment: Is `gas - 21` actually still usable?

**Review by chriseth — APPROVED** [review]

File: liblll/CompilerState.cpp (hunk 1 above)
Comment (zigguratt) [reply liblll/CompilerState.cpp:48]: Well in my own code I've been using `(- (gas) 1000)` to be safe. We could definitely bump it up in `CompilerState.cpp` at some point.

Comment (chriseth) [reply liblll/CompilerState.cpp:48]: In a few days, you can always just use `gas`, so I think it is not relevant anymore anyway.

**Comment by chriseth** [comment]
Please rebase, it is impossible to review merge commits.

**Comment by axic** [comment]
Thanks @zigguratt for the PR! :tada:

**Comment by zigguratt** [comment]
@axic The first of many to come, I hope.

**Pull request merged**